### ISSUE TITLE
Support all accessibilityRoles

### ISSUE THREAD

**Issue by pjivers-bom** [FormidableLabs/eslint-plugin-react-native-a11y#56]
Current accessibility roles, such as `'switch'`, should be supported: https://facebook.github.io/react-native/docs/accessibility.html#accessibility-properties

**Comment by jpdriver** [comment]
fixed with [v1.3.0](https://github.com/FormidableLabs/eslint-plugin-react-native-a11y/releases/tag/v1.3.0)
